package.json

```diff
@@ -1,130 +1,35 @@
 {
-  "name": "@bluecadet/launchpad",
-  "version": "1.0.4",
-  "description": "Suite of tools to manage media installations",
-  "engines": {
-    "npm": ">=7"
-  },
-  "type": "module",
-  "main": "./index.js",
-  "bin": {
-    "launchpad": "./index.js"
-  },
+  "name": "launchpad",
```

Comment:
@claytercek do you know how this will interact with NPM?

So if we want to install launchpad, we'll still do `npm i @bluecadet/launchpad`? Will that pull `launchpad-core` and conversely all the dependencies?

What about the version number, visibility and description? Are all of those just for GH then?

Reply:
Yep! This whole `package.json` is only relevant for development, mostly just to tell npm where the workspaces are. We can put whatever we want for name, description, and version because it'll never be published on any registry.

Reply:
Actually, given that that's the case, do you think it would make sense to rename the `core` directory to `launchpad`?

Reply:
Yeah I think that makes sense. Then makes me wonder if all the other dirs should be prefixed with `launchpad-` too, but that feels verbose 🤔 Let's just do `launchpad` for `core` fore now. This can def wait until tomorrow!

Reply:
updated in 15cd223